addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -924,3 +929,9 @@ msgstr ""
 msgctxt "#31611"
 msgid "System"
 msgstr ""
+
+#: /xml/Includes_DialogSelect.xml
```

Comment:
We use the fully qualified name here => addons/skin.estuary/xml...
I know this is not used correctly at other places in this file, but for new strings we should do it right.

Reply:
on kodi language files yes but i have never seen this on skin language files,
its a new rule?

Reply:
If you search for addons/skin.estuary/ in this file you will get many hits.

Reply:
Oh wait, you are right. This is strings.po of the estuary skin, not the global strings.po. Sorry.